ansys/mapdl/core/misc.py

```diff
@@ -333,3 +333,54 @@ def wrapper(*args, **kwargs):
         func(*args, **kwargs)
 
     return wrapper
+
+
+# Function to build main
+def check_res_table(row_names=None, simulation_res=None, target_res=None):
```

Comment:
If these parameters are always required, then should be using arg and not kwarg.